<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionSetupDialog</class>
 <widget class="QDialog" name="SessionSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="accessibleName">
   <string>Orbit Profiler</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
	background-image: url(:/actions/search_small_offset);
	background-position: left center;
	background-repeat: no-repeat;
	height: 22px;
	padding-left: 20px;
	padding-bottom: 1px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>6</number>
       </property>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="orbit_session_setup::ConnectToLocalWidget" name="localProfilingWidget" native="true"/>
         </item>
         <item>
          <widget class="orbit_session_setup::LoadCaptureWidget" name="loadCaptureWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="orbit_session_setup::ProcessListWidget" name="processListWidget" native="true"/>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(68, 68, 68)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="orbit_session_setup::TargetLabel" name="targetLabel" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="confirmButton">
          <property name="accessibleName">
           <string>StartOrbit</string>
          </property>
          <property name="text">
           <string>Start Session</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>orbit_session_setup::LoadCaptureWidget</class>
   <extends>QWidget</extends>
   <header>SessionSetup/LoadCaptureWidget.h</header>
  </customwidget>
  <customwidget>
   <class>orbit_session_setup::TargetLabel</class>
   <extends>QWidget</extends>
   <header>SessionSetup/TargetLabel.h</header>
  </customwidget>
  <customwidget>
   <class>orbit_session_setup::ConnectToLocalWidget</class>
   <extends>QWidget</extends>
   <header>SessionSetup/ConnectToLocalWidget.h</header>
  </customwidget>
  <customwidget>
   <class>orbit_session_setup::ProcessListWidget</class>
   <extends>QWidget</extends>
   <header>SessionSetup/ProcessListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/orbiticons.qrc"/>
 </resources>
 <connections/>
</ui>
